experimental/Schemes/Types.jl

```diff
@@ -513,30 +434,59 @@ identifications given by the glueings in the `default_covering`.
         end
         return hom(OV, OU, rho_func.(gens(OV)), check=false)
       else
-        G = default_covering(X)[ambient_scheme(V), U]
+        G = default_covering(X)[W, U]
         W1, W2 = glueing_domains(G)
         f, g = glueing_morphisms(G)
+        g_res = restrict(g, U, V_direct)
+        inc_res = restrict(incV, V, V_direct, check=false)
+        return pullback(compose(g_res, inverse(inc_res)))
+        ### deprecated code below
```

Comment:
If it is deprecated, why don't you put in in a comment? Is it still in use somewhere?